stock_picking_invoice_product_group/models/account_invoice.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# (c) 2015 Alex Comba - Agile Business Group
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import models, fields
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    related_picking_ids = fields.Many2many(
```

Comment:
I don't agree. Having each module solving by their own, you will end up with three fields for the same purpose. The base module must handle all possible cases from Odoo core + OCA. Now our ecosystem should be expanded not only for the core. The DB change is something that we can perfectly deal with migration scripts. We have done it previously on lot of OCA modules.